person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..10_000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    @parent_permission || is_of_age?
+  end
+
+  private
+
+  def of_age?
+    @age > 18
```

Comment:
- [ ] Age must be greater or equal to the 18. Kindly fix this issue.

Reply:
Hello @Rustamxon7 😊
Thanks for your feedback,
I did the changes which you stated, you can check them now